cinema_service/settings.py

```diff
@@ -24,7 +24,7 @@
     "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
 )
 
-# SECURITY WARNING: don't run with debug turned on in production!
+# SECURITY WARNING: don"t run with debug turned on in production!
 DEBUG = True
```

Comment:
Set `DEBUG = False` for production environments to prevent the exposure of sensitive information.